catalog/dags/providers/provider_api_scripts/rawpixel.py

```diff
@@ -258,9 +282,14 @@ def get_record_data(self, data: dict) -> dict | list[dict] | None:
         if not (license_info := get_license_info(metadata["licenseUrl"])):
             return None
 
-        if not (url := self._get_image_url(data, self.full_size_option)):
+        full_size_preset = self._get_full_size_preset(self, data)
+        if not (url := self._get_image_url(data, full_size_preset)):
             return None
 
+        thumbnail_preset = self._get_thumbnail_size_preset(self, data)
+        if not (thumbnail := self._get_image_url(data, thumbnail_preset)):
+            thumbnail = url
```

Comment:
For thumbnails, we actually have a `thumbnail_url` field that can be used directly!

https://github.com/WordPress/openverse/blob/614720f54a9510261b2dede5ddd0a53f51a2fb73/catalog/templates/template_provider.py_template#L141

This would be better served as a top level field alongside `meta_data` (rather than inside it), so that it can be used as the thumbnail in our frontend. Do you mind moving it there? I think we'd also prefer to leave thumbnail as `None` if it doesn't exist, rather than setting it to the main image URL.

Reply:
done ✔️